docs/static/breaking-changes.asciidoc

```diff
@@ -12,6 +12,21 @@ after installing it via a package (RPM, DEB).
 respectively. Previously it used to install in `/opt` directory. This change was done to make the user experience 
 consistent with other Elastic products. Full directory layout is described in <<dir-layout>>.
 
+**Default Logging Level:** Changed the default log severity level to INFO instead of WARN to match Elasticsearch. Existing logs
+(in core and plugins) were too noisy at INFO level, so we had to audit log messages and switch some of them to DEBUG
+level.
+
+**Index Template:** The index template for 5.0 has been changed to reflect Elasticsearch's mapping changes. Most
```

Comment:
Can we link `Elasticsearch's mapping changes` to https://www.elastic.co/guide/en/elasticsearch/reference/5.0/breaking_50_mapping_changes.html#_literal_string_literal_fields_replaced_by_literal_text_literal_literal_keyword_literal_fields